components/CovidMap.js

```diff
@@ -0,0 +1,702 @@
+  height: '35', 
+  className: 'custom-clustericon-first'
+}
+
+const clusterStyleFirstLarge = {
+  width: '40', 
+  height: '40', 
+  className: 'custom-clustericon-first'
+}
+
+const clusterStyleSecondSmall = {
+  width: '30', 
+  height: '30', 
+  className: 'custom-clustericon-second'
+}
+
+const clusterStyleSecondMed = {
+  width: '35', 
+  height: '35', 
+  className: 'custom-clustericon-second'
+}
+
+const clusterStyleSecondLarge = {
+  width: '40', 
+  height: '40', 
+  className: 'custom-clustericon-second'
+}
+
+const clusterStyleThirdSmall = {
+  width: '30', 
+  height: '30', 
+  className: 'custom-clustericon-third'
+}
+
+const clusterStyleThirdMed = {
+  width: '35', 
+  height: '35', 
+  className: 'custom-clustericon-third'
+}
+
+const clusterStyleThirdLarge = {
+  width: '40', 
+  height: '40', 
+  className: 'custom-clustericon-third'
+}
+
+const formStyle = {
+  width: '50%',
+  float: 'left', 
+}
+
+const legendStyle = {
+  width: '50%', 
+  float: 'right',
+}
+
+const legendItemStyle = {
+  width: '100%',
+  marginBottom: '5px',
+}
+
+const legendTextStyle = {
+  width: '90%',
+  height: '20px',
+  float: 'left',
+  textAlign: 'right',
+}
+
+const legendIconStyle = {
+  minWidth: '30px',
+  height: '20px',
+  float: 'right',
+  display: 'inline-block',
+}
+
+const legendIconFirst = {
+  background: '#0B5D93',
+  width: '20px',
+  height: '20px',
+  borderRadius: '50%',
+  display: 'block',
+  float: 'right',
+  marginLeft: '5px',
+  marginRight: '5px',
+  opacity: '80%',
+}
+
+const legendIconSecond = {
+  background: '#634562',
+  width: '20px',
+  height: '20px',
+  borderRadius: '50%',
+  display: 'block',
+  float: 'left',
+  marginLeft: '5px',
+  marginRight: '5px',
+  opacity: '80%',
+}
+
+const legendIconThird = {
+  backgroundColor: '#CE2727',
+  width: '20px',
+  height: '20px',
+  borderRadius: '50%',
+  display: 'block',
+  float: 'left',
+  marginLeft: '5px',
+  marginRight: '5px',
+  opacity: '80%',
+}
+
+
+class Map extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      map: null,
+      firstClusterer: null,
+      secondClusterer: null, 
+      thirdClusterer: null,
+      fetchedMap: false,
+      data: [],
+      selectUpdate: true,
+      selectedOption: props.value || 'all',
+      firstLegendText: 'All Deaths',
+      secondLegendText: '',
+      thirdLegendText: ''
+    }
+  }
+
+  getGoogleMaps() {
+    // If we haven't already defined the promise, define it
+    if (!this.googleMapsPromise) {
+      this.googleMapsPromise = new Promise((resolve) => {
+        // Add a global handler for when the API finishes loading
+        window.resolveGoogleMapsPromise = () => {
+          // Resolve the promise
+          const google = window.google;
+          resolve(google);
+
+          // Tidy up
+          delete window.resolveGoogleMapsPromise;
+        };
+
+        // Load the Google Maps API
+        const API = 'AIzaSyDAh7M89BnID8kGVXBrNtxJfD-jjDDFRCg';
```

Comment:
Whose Google Maps API key is this? Is it safe to include in our open source code? Is it [restricted](https://developers.google.com/maps/documentation/javascript/get-api-key) in any way?

Reply:
No, it needs to be stored and shouldn't be checked in here. It's just one I created for this application and it has yet to be restricted. I wasn't sure if my own key will work, or if perhaps TJI already had one, or perhaps we need to create a Google account for TJI so we can generate it's own google key?

Reply:
Sounds like @LayaTaal is working with you to store this API key in Netlify. Though, for what it's worth, no matter where we store this key, it's going to be exposed in client-side API requests.

I think it would be a good idea to [restrict this key](https://developers.google.com/maps/documentation/javascript/get-api-key#restrict_key) to some specific domains for production and deploy preview apps so that we don't get any billing surprises if a bad actor gets ahold of it. TJI has a Google account, and ideally we'd manage the API key there, but I'd be happy with handling that as a follow up to this PR.